specs/commandline/key.md

```diff
@@ -115,10 +115,27 @@ notation key list
 
 Upon successful execution, a list of keys is printed out with information of name, key path, certificate path, key id and plugin name. The default signing key name is preceded by an asterisk. The key id and plugin name are used together to provide the information of the key identifier for the remote key and the plugin associated with it.
 
-### Delete two keys from signing key list
+### Remove a specified key from signing key list
 
 ```shell
-notation key delete <key_name_1> <key_name_2>
+notation key remove <key_name>
 ```
 
-Upon successful execution, the names of deleted signing keys are printed out. Please be noted if default signing key is deleted, Notation will not automatically assign a new default signing key. User needs to update the default signing key explicitly.
+- Upon successful execution, the output message is printed out as below when removed a key created by notation for local testing purpose:
+
+```text
+Removed <key_name> from signing key list. You still need to delete the key file from <key_path> and delete the certificate from <cert_path>.
+```
+- Upon successful execution, the output message is printed out as below when removed a key associates with a KMS:
+
+```text
+Removed <key_name> from signing key list.
+```
+
+### Remove two keys from signing key list
+
+```shell
+notation key remove <key_name_1> <key_name_2>
+```
+
+Upon successful execution, the names of removed signing keys are printed out. Please be noted if default signing key is removed, Notation will not automatically assign a new default signing key. User needs to update the default signing key explicitly.
```

Comment:
do we need an example output for removing multiple keys? even if it's just
```shell
key_name_1
key_name_2
```

Reply:
Accepted and added a new output example for removing multiple keys.